sources/packages/backend/libs/sims-db/src/entities/notification.model.ts

```diff
@@ -136,4 +136,8 @@ export enum NotificationMessageType {
    * MSFAA record gets cancelled.
    */
   MSFAACancellation = 12,
+  /**
+   *  An Application Offering Change Request is in progress with the student.
+   */
+  OfferingChangeRequestInProgressWithStudent = 13,
```

Comment:
https://github.com/bcgov/SIMS/pull/2114/files#r1264232174

Reply:
If there is a common template for all 3 scenarios, I will update the names everywhere.